app/code/Magento/Sales/view/frontend/templates/email/items/order/default.phtml

```diff
@@ -21,7 +21,14 @@ $_order = $_item->getOrder();
             <?php foreach ($block->getItemOptions() as $option) : ?>
                 <dt><strong><em><?= $block->escapeHtml($option['label']) ?></em></strong></dt>
                 <dd>
-                    <?= /* @noEscape */  nl2br($block->escapeHtml($option['value'])) ?>
+                    <?php
+                    if ($option['option_type'] == 'file') {
```

Comment:
Should be \Magento\Catalog\Api\Data\ProductCustomOptionInterface::OPTION_TYPE_FILE

Reply:
I have applied changes as per suggestion as well as I have tried and fixed some phpcs errors.